setup.py

```diff
@@ -43,7 +43,7 @@ def package_files(package_dir, subdirectory):
                 'scripts/query_ex_ants.py', 'scripts/red_average.py',
                 'scripts/xtalk_filter_run.py', 'scripts/time_average.py',
                 'scripts/time_chunk_from_baseline_chunks_run.py', 'scripts/chunk_files.py',
-                'scripts/flag_all.py', 'scripts/throw_away_flagged_antennas.py'],
+                'scripts/flag_all.py', 'scripts/throw_away_flagged_antennas.py', 'scripts/select_spw_ranges_run.py'],
```

Comment:
I'd probably remove the word `_run` since that's most there to distinguish this from a module filename and that's not a problem here.

Reply:
I've removed `_run`.